musicrs/model/base.py

```diff
@@ -0,0 +1,33 @@
+""" Base models of the database tables """
+
+from sqlalchemy import Column, Integer, String, TIMESTAMP, func
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+
+class UserProfile(Base):
```

Comment:
We agreed on creating different model for user also.Aren't we doing it?

Reply:
Yes, we did agree on that.. However, we are unsure of its actual pros and cons. So, lets iterate on this model and make changes when required. This is the base model to work on.